alfresco-search/src/test/java/org/alfresco/solr/query/afts/requestHandler/AFTSDefaultTextQueryTest.java

```diff
@@ -0,0 +1,163 @@
+    public static void beforeClass() throws Exception
+    {
+        TestDataProvider dataProvider = new TestDataProvider(h);
+
+        String[] names = {
+                "test1",
+                "test2",
+                "file3",
+                "name4"};
+
+        String[] descriptions = {
+                "description of test 1",
+                "description 2",
+                "this is not a description of test 1 and 2",
+                "other description"};
+
+        String[] contents = {
+                "test",
+                "content of test 2",
+                "other contents",
+                "content of file number 4"};
+
+        String[] titles = {
+                "Title1",
+                "Other Title",
+                "Third",
+                "Forth"};
+
+        // Creators should not be searched in default queries
+        String[] creators = {
+                "Luca",
+                "Mario",
+                "Giovanni",
+                "Giuseppe"};
+
+        // Make sure the arrays of properties have the same cardinality.
+        assertEquals(names.length, descriptions.length);
+        assertEquals(names.length, contents.length);
+        assertEquals(names.length, titles.length);
+        assertEquals(names.length, creators.length);
+
+        TEST_ROOT_NODEREF = dataProvider.getRootNode();
+
+        range(0, names.length)
+                .forEach(dbId -> {
+                    String name = names[dbId];
+                    String description = descriptions[dbId];
+                    String content = contents[dbId];
+                    String title = titles[dbId];
+                    String creator = creators[dbId];
+
+                    Map<QName, PropertyValue> properties = new HashMap<>();
+                    properties.put(PROP_NAME, new StringPropertyValue(name));
+                    properties.put(PROP_DESCRIPTION, new StringPropertyValue(description));
+                    properties.put(PROP_CONTENT, new StringPropertyValue(content));
+                    properties.put(PROP_TITLE, new StringPropertyValue(title));
+                    properties.put(PROP_CREATOR, new StringPropertyValue(creator));
+
+                    addNode(getCore(),
+                            dataModel, 1, dbId, 1,
+                            TYPE_CONTENT, null, properties, null,
+                            "the_owner_of_this_node_is" + name,
+                            null,
+                            new NodeRef[]{ TEST_ROOT_NODEREF },
+                            new String[]{ "/" + dataProvider.qName("a_qname_for_node_" + name) },
+                            dataProvider.newNodeRef(), true);
+                });
+    }
+
+    @Test
+    public void defaultExactQueryTest()
+    {
+        assertResponseCardinality("test", 3);
+        assertResponseCardinality("Giovanni", 0);
+    }
+
+    @Test
+    public void defaultRangeQueryTest()
+    {
+        assertResponseCardinality("[te to test]", 3);
```

Comment:
This is currently matching "test" three times. We could instead use a range "[test0 to test9]" which should match two docs.  It might be worth mentioning which docs we're expecting to match too.

Reply:
I added some comments in order to explain the expected results and I improved the coverage of the tests. The goal of the tests is to perform different kind of queries and see if the fields involved are the right ones. For instance, I want to check that range queries actually works on cm:name, cm:title, cm:description and cm:content but not the correctness of range queries.

Reply:
I think this pair of tests currently doesn't check if the name or the creator fields are checked.